test/saucelabs/print_test.py

```diff
@@ -0,0 +1,168 @@
+import time
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+
+DEFAULT_WAIT = 10
+QUERYSTRING_KML = 'KML%7C%7Chttp:%2F%2Fopendata.utou.ch%2Furbanproto%2Fgeneva%2Fgeo%2Fkml%2FRoutes.kml'
```

Comment:
use quote from urllib2 if possible and put something human readable (some humans only=)